### PR TITLE
[Term Entry] ai neural networks: Added detailed description on Stochastic gradient …

### LINKED ISSUE #4527: [Term Entry] Neural Networks ai stochastic gradient descent
### Reason/inspiration (optional)

"We would like a new term entry in the `AI` concept for [neural-network](https://www.codecademy.com/resources/docs/ai/neural-networks):  stochastic gradient descent. The entry should go in a new file under `docs/content/ai/concepts/neural-networks/terms/stochastic-gradient-descent/stochastic-gradient-descent.md`.

The entry should describe the term and how it is used. Also include: 
- An "Example" section that explains to readers how to use concept for stochastic gradient descent.
- Should contain information for this concept and syntax.
- Should have a Python codebyte with a working code snippet showcasing this concept.
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by TaylaCorney**
### Description
Added a baseline description of stochastic gradient decent to be added into the neural network docs under ai. Having looked at the other documentation inside neural networks, the level of detail which was applied was similar to that of the other docs. 

### Issue Solved

Closes #4527 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

**Comment by cigar-galaxy82** [comment]
Hi @TaylaCorney can you please fix this PR or re-open the PR running on a new branch if you can't fix the issue with the current branch

**Review by mohitO12-dev — COMMENTED** [review]
invalid
